packages/uicore/src/components/MultiSelectDropDown/MultiSelectDropDown.tsx

```diff
@@ -49,6 +49,7 @@ export interface MultiSelectDropDownProps
   placeholder?: string
   hideItemCount?: boolean
   onPopoverClose?(opts: MultiSelectOption[]): void
+  showBaseInputBlue?: boolean
```

Comment:
What is `showBaseInputBlue` supposed to do? Not an intuitive props.

Reply:
Remove this and use `className` props for any additional styling

Reply:
resolved